owl/SOMA-ACT.owl

```diff
@@ -198,14 +198,14 @@ Declaration(NamedIndividual(SOMA:ExecutionState_Succeeded))
 #   Object Properties
 ############################
 
-# Object Property: SOMA:affects (affects)
+# Object Property: <http://www.ease-crc.org/ont/SOMA.owl#affects> (affects)
```

Comment:
A lot of the file difference goes back to this version of functional syntax not abbreviating the IRIs correctly. If fixed, a Review would be much easier.
I hope that it would be enough to set a PrefixManager for the OWLDocumentFormat.

Reply:
It might otherwise help to load the ontologies in protege and then save them again as functional syntax as this might also be due to implementation differences in protege vs. owl api